spanner/cloud-client/src/main/java/com/example/spanner/LoadCsvExample.java

```diff
@@ -0,0 +1,271 @@
+/*
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner;
+
+// Imports the Google Cloud client library
+import com.google.cloud.spanner.Mutation;
+import com.google.cloud.spanner.Mutation.WriteBuilder;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.cloud.spanner.jdbc.CloudSpannerJdbcConnection;
+import java.io.FileReader;
+import java.io.Reader;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.Options;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+
+/** Sample showing how to load CSV file data into Spanner */
+public class LoadCsvExample {
+  public static final String EXCEL = "EXCEL";
+  public static final String POSTGRESQL_CSV = "POSTGRESQL_CSV";
+  public static final String POSTGRESQL_TEXT = "POSTGRESQL_TEXT}";
+
+  enum SpannerDataTypes {
+    STRING,
+    INT64,
+    FLOAT64,
+    BOOL,
+    DATE,
```

Comment:
We aren't supporting Date right now, but we are supporting Bytes. Can you remove DATE and add BYTES?

Reply:
According to the starter project documentation, I thought we are supporting DATES and not BYTES. Not sure if I'm missing something of if this has changed since the starter project write up was written.

Reply:
since Diem is out today, let's just go ahead and include all the Spanner types, as you have done